### PR TITLE
Update how_this_course_will_work.md

### PR DESCRIPTION
**Pull request by fortypercenttitanium**
Added a section addressing non-English speakers

 - [X] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

Added a section in the intro to help non-English speakers be prepared for the course by telling them it's written by and maintained by English speakers. Recommends they use a translation dictionary and utilize additional resources in their own language, as well as spend extra time on things they don't understand.

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

**Review by I3uckwheat — CHANGES_REQUESTED** [review]
Could we add a bit on the fact we won't translate TOP as well?

**Comment by marvingay** [comment]
I'm going to leave a small review shortly.

File: foundations/the_basics/how_this_course_will_work.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ This curriculum works by pulling together the best content from across the inter
 
 Most lessons will contain questions that you should be able to answer before moving on.  Some of the lessons will ask you to complete exercises. In addition, we provide several projects throughout the curriculum to help you grow your understanding by actually building things.
 
+### A Note About Language
+
+The Odin Project attracts people from all over the world who aspire to learn how to program. Please be aware that this curriculum is written in English and maintained by people whose primary (and typically only) language is English. If you are a non-English speaker, or English is not a primary language for you, we *highly* recommend spending extra time on topics and terms that you do not understand right away. We also encourage you to seek out additional resources that teach about these topics in your own native language so you can understand them more fully.
```
Comment (marvingay) [inline foundations/the_basics/how_this_course_will_work.md:25]: It's imperative to point out somewhere here why English is important to programming at-large.

(e.g. official documentation/references/and the community at-large [Odin and beyond] speak or read it).

Other than that, seems good.

**Review by marvingay — CHANGES_REQUESTED** [review]

**Review by fortypercenttitanium — COMMENTED** [review]
Made requested changes. Let me know if you like.

**Review by marvingay — APPROVED** [review]
Looks good to me.

**Review by I3uckwheat — APPROVED** [review]

**Pull request merged**